src/torch_models.py

```diff
@@ -437,6 +435,30 @@ def _fit(self, inputs: Array, outputs: Array) -> None:
         print(f"Loaded best model with loss: {loss:.5f}")
 
 
+class MLPClassifierEnsemble(Classifier):
+    """MLPClassifierEnsemble definition."""
+
+    def __init__(self, in_size: int, max_itr: int, n: int) -> None:
+        self._members = [
+            MLPClassifier(in_size, max_itr, CFG.seed + i) for i in range(n)
+        ]
+
+    def fit(self, X: Array, y: Array) -> None:
+        for i, member in enumerate(self._members):
+            print(f"Fitting member {i} of ensemble...")
+            member.fit(X, y)
+
+    def classify(self, x: Array) -> bool:
+        member_vals = []
+        for member in self._members:
+            x_normalized = member._normalize(x)  # pylint: disable=protected-access
```

Comment:
any reason we don't want to make this public?

Reply:
oh sorry i made that change and hadn't pushed yet